slides/cen-first-year-wg-sep2023.qmd

```diff
@@ -0,0 +1,209 @@
+
+## Why was the MMRM topic important?
+
+- MMRM is a popular analysis of longitudinal continuous outcomes in randomized clinical trials
+- No tailored R package with sufficient capabilities/reliability
+- Also used as backbone for more recent methods such as multiple imputation
+- Have a look at Gonzalo’s presentation to learn more about `mmrm`
+  - 10.40 am in the following session 
+  - compares `mmrm` with previous R packages and SAS
+
+## Best practices
+
+- Includes version control, git workflow, code review, unit testing, integration tests, continuous deployment, reproducibility, package design, object oriented programming, versioning, publication, etc.
+- Workshop "Good Software Engineering Practice for R Packages" on world tour
+  - Basel, Shanghai, San José, Rockville MD, Montreal [🌎](https://openpharma.github.io/workshop-r-swe-mtl/)
+- Start of video series "Statistical Software Engineering 101" [🌎](https://www.youtube.com/playlist?list=PL848NFA2PWgCR35n02yn1ZV7JqSu3NMxS)
+  - currently 2 videos, hopefully we can still produce more content
+
+## Conference contributions and Publications
+
+- Dedicated sessions with discussions at ISCB, CEN (now), ASA/FDA workshop
+- Presentations at PSI, JSM, Pharma RUG, BBS, etc. [🌎](https://rconsortium.github.io/asa-biop-swe-wg/presentations.html)
+- BIOP Report [🌎](https://rconsortium.github.io/asa-biop-swe-wg/blog/biop_report/)
+- Blog [🌎](https://rconsortium.github.io/asa-biop-swe-wg/blog.html)
+
+# Ingredients for successful and sustainable collaboration
+
+## Human factors
+
+- Mutual interest
+- Mutual trust
+- Prerequisite is getting to know each other
+  - Although mostly just online, biweekly calls help a lot with this
+- Reciprocity mindset
+  - First give away something, and more will come back to you
+
+## Development process
+
+- Different perspectives in discussions and code review help to optimize the user interface and thus experience
+- Important to go public as soon as possible 
+  - don't wait for the product to be finished
+  - you never know who else might be interested/could help
+- Version control with git
+  - cornerstone of effective collaboration
+
+## Coding standards
+
+- Consistent and readable code style simplifies joint work
+- Written (!) contribution guidelines
+- Lowering the entry hurdle using developer calls
+
+## Robust test suite
+
+- Unit tests are essential for preventing regression and assuring quality
+- Especially with compiled code critical to see if package works correctly
+- Use continuous integration during development to make sure nothing breaks along the way
+
+## Documentation
+
+- Lots of work but extremely important 
+  - often need to start with methods details write up before being able to code
+- Needs to be kept up-to-date
+- Need to have examples & vignettes
+  - Testing alone is not sufficient 
+  - Builds trust with users
+  - Reference for developers over time
+
+# Long term vision
+
+## Statisticians have software engineering skills
```

Comment:
Does this refer to the long term vision or to the current status?

Reply:
ah yeah, I added "Vision:" now to clarify